src/java/org/apache/cassandra/db/virtual/SystemPropertiesTable.java

```diff
@@ -35,6 +35,7 @@
     private static final Set<String> CASSANDRA_RELEVANT_PROPERTIES = Sets.newHashSet(
             // base jvm properties
             "java.home",
+            "JAVA_HOME",
```

Comment:
instead of adding `JAVA_HOME` here, what do you think about adding it to a second set `CASSANDRA_RELEVANT_ENVS` ?  (even if it's a singleton, maybe it's cleaner code, and allows adding more env more easily…?)

Reply:
Agreed, done, thanks